### DISCUSSION TITLE
Stop lowest map layer being removed from map

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1445] [Ideas]
**Is your feature request related to a problem? Please describe.**
In #798 a fix was added to skip map layer that's don't have data. This means that no matter how far you zoom into the lowest layer of a map, that layer doesn't get removed. 

However, if data is available at highest level e.g. country but then not available at lower levels e.g. region, the country level data is removed from the map. This looks quite strange. See [here](https://sdgdata.gov.uk/3-a-1/) for example

**Describe the solution that you need**
Lowest layer of map in each area stays showing when zoomed in,

**Why do you need this feature?**
This was suggested by Geography team at ONS when discussing maps with them - it looks better that way. 

**Describe alternatives you've considered**
None

**Comment by brockfanning** [comment]
Possible solution: add additional "features" to the geojson file so that the appear on multiple zoom levels.

**Reply by LucyGwilliamAdmin** [reply]
@brockfanning I've tried this and the geojson seems to look fine: https://github.com/ONSdigital/sdg-data/blob/feature-geojson-test/Regions-and-Countries.geojson

But then when you actually view the map on the feature branch, it doesn't seem to work:
http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-geojson-test/3-a-1/
